examples/kendo-react-freemium/package.json

```diff
@@ -0,0 +1,82 @@
+{
+    "name": "kendo-react-freemium",
+    "private": true,
+    "version": "0.0.0",
+    "homepage": "https://telerik.github.io/kendo-react/kendo-react-freemium/",
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "tsc -b && vite build",
+        "preview": "vite preview",
+        "predeploy": "npm run build",
+        "deploy": "gh-pages -d build"
+    },
+    "dependencies": {
+        "@progress/kendo-data-query": "^1.7.0",
+        "@progress/kendo-drawing": "^1.20.2",
+        "@progress/kendo-licensing": "^1.3.5",
+        "@progress/kendo-react-animation": "^9.5.0-develop.1",
```

Comment:
licensing is now at [1.5.0](https://github.com/telerik/kendo-licensing/releases/tag/v1.5.0) and the kendo react products can be updated to 9.5.0 as we have an official version

Reply:
Also a note -> when we release freemium, the version will be bumped to 10.0.0, so we need to update it here as well